model/bin/cmplr.env

```diff
@@ -189,11 +174,19 @@ if [ "$cmplr" == "intel" ]          || [ "$cmplr" == "intel_debug" ]    || [ "$c
   if [ ! -z "$(echo $cmplr | grep datarmor)" ] ; then
     optc="$optc -xcore-avx2"
     optl="$optl -xcore-avx2"
+  elif [ ! -z "$(echo $cmplr | grep wcoss_cray)" ] ; then
```

Comment:
we should probably do some sanity checks by running ufs-weather-model on wcoss @aliabdolali

Reply:
Probably a sensible thing to do!
A good first check is to make sure the list of "opts" in your generated `comp` and `link` scripts are the same...

Reply:
checked on WCOSS successfully.